src/components/list/ListForm.vue

```diff
@@ -232,79 +277,95 @@ export default {
         target="_blank"
       />
     </div>
-    <div class="grouping-fields-container">
-      <div class="tags">
-        <SectionCard
-          title="tags"
-          :size="isMobileScreen ? 'small' : ''"
-        >
+    <SectionCard
+      title="tags"
+      position="left"
+      text-style="caps"
+      size="small"
+    >
+      <MultiselectCustom
+        ref="tags"
+        :value="tagsTitles"
+        :options="tagsTitles"
+        mode="tags"
+        placeholder="type and press Enter to add new tag"
+        :can-clear="false"
+        :create-option="!titleErrors.tags"
+        :show-options="!titleErrors.tags"
+        :disabled="requestHandling.isRequestProcessing"
+        no-options-text=""
+        no-results-text=""
```

Comment:
for what is it here like that?

Reply:
initially, this multiselect component is designed for choosing options from dropdown
and when there are no options to choose from, it shows some no-options-text by default
no-results-text is also shown by default when creation options is turned off and nothing found in search

but tags and categories in list form have a bit different concept: they are NOT chosen from dropdown, they are just added anew, so these two props should be empty, otherwise some default text will be shown